core/SConstruct

```diff
@@ -1,3 +1,5 @@
+CacheDir('build_cache')
```

Comment:
Can we make it so that `make clean` wipes the cache? I.e. put it in `build/cache` or update Makefile to delete this directory too.